nixos/modules/services/web-servers/garage.nix

```diff
@@ -60,7 +60,7 @@ in
 
     package = mkOption {
       # TODO: when 23.05 is released and if Garage 0.9 is the default, put a stateVersion check.
-      default = if versionAtLeast stateVersion "23.05" then pkgs.garage_0_8_0
+      default = if versionAtLeast config.system.stateVersion "23.05" then pkgs.garage_0_8
```

Comment:
No, it should use 0.8.0 and not 0.8.x.
Good catch for `stateVersion`, thank you!

Reply:
Ugh, 0.8.0 disappeared, hm.

Reply:
Okay, let's get this then.

Reply:
Yeah I wasn't sure what was going on with that haha. Do you want 0_8_0 back?

Reply:
Also I'm surprised, I thought this module was covered by tests?

Reply:
No no, no worries. It's just this 0.x versioning makes me confused sometimes whether 0.8 -> 0.9 is like 8.0 -> 9.0 or 0.8.0 -> 0.8.1 is 8.1 -> 8.2 which is a minor bump and all that stuff.
We are fine with 0.8.x in fact. Apologies (I'm tired :D).

Reply:
> Also I'm surprised, I thought this module was covered by tests?

Nix is lazy, stateVersion is unchecked then